--- OpenSP-1.5.1/configure.in.orig	2003-10-22 22:23:52.639805072 +0200
+++ OpenSP-1.5.1/configure.in	2003-10-22 22:26:22.293054320 +0200
@@ -242,7 +242,7 @@
 ALL_LINGUAS="de fr ja sv"
 AM_GNU_GETTEXT
 AC_CHECK_HEADERS(locale.h)
-AC_DEFINE_DIR(SP_LOCALE_DIR, datadir/locale, [location of message catalogs])
+AC_DEFINE_UNQUOTED(SP_LOCALE_DIR, "${prefix}/share/locale", [location of message catalogs])
 SP_MESSAGE_DOMAIN="sp$SP_LIBOSP_CUR"
 AC_DEFINE_UNQUOTED(SP_MESSAGE_DOMAIN, "$SP_MESSAGE_DOMAIN", [message domain])
 AC_SUBST(SP_MESSAGE_DOMAIN)
