diff -urNp -x '*.orig' OpenSP-1.5.2.org/configure.in OpenSP-1.5.2/configure.in
--- OpenSP-1.5.2.org/configure.in	2023-12-03 23:23:38.606960871 +0100
+++ OpenSP-1.5.2/configure.in	2023-12-03 23:23:38.684961806 +0100
@@ -241,7 +241,7 @@ AC_CHECK_LIB(pthread,pthread_create,,AC_
 AM_GNU_GETTEXT
 AM_GNU_GETTEXT_VERSION([0.14.5])
 AC_CHECK_HEADERS(locale.h)
-AC_DEFINE_DIR(SP_LOCALE_DIR, datadir/locale, [location of message catalogs])
+AC_DEFINE_UNQUOTED(SP_LOCALE_DIR, "${prefix}/share/locale", [location of message catalogs])
 SP_MESSAGE_DOMAIN="sp$SP_LIBOSP_CUR"
 AC_DEFINE_UNQUOTED(SP_MESSAGE_DOMAIN, "$SP_MESSAGE_DOMAIN", [message domain])
 AC_SUBST(SP_MESSAGE_DOMAIN)
